server/templates/obslist.html

```diff
@@ -103,6 +103,17 @@ <h3>Observations</h3>
                     {{ obs.station_name }}
                 </a>
             </td>
+            <td>
+                {% if obs.config.frequency is defined %}
+                {{ "%.3f MHz"|format(obs.config.frequency/1000000) }}
```

Comment:
We will need a special frequency helper here (not necessarily now). The frequencies below 1MHz should be formatted in Hz, above 1GHz in GHz.

Reply:
Frequencies below 1MHz are useless for satellites, but I get your point. I'm not enthusiastic: mixing MHz and GHz may mess up sorting, but I can see how it could improve readability.